package.json

```diff
@@ -6,43 +6,43 @@
     "start": "ng serve",
     "build": "ng build",
     "test": "ng test",
-    "lint": "ng lint",
+    "lint": "gts lint",
     "e2e": "ng e2e",
     "check": "gts check",
     "clean": "gts clean",
-    "compile": "tsc -p .",
+    "compile": "tsc",
     "fix": "gts fix",
     "prepare": "npm run compile",
     "pretest": "npm run compile",
-    "posttest": "npm run check"
+    "posttest": "npm run lint"
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "~8.2.0",
+    "@angular/animations": "~10.1.5",
     "@angular/cdk": "^8.1.2",
```

Comment:
i think you need to `ng upgrade @angular/cdk` as well, it's not part of the default upgrade group, but it does use the angular release cycle